packages/manager/apps/pci-cold-archive/src/pages/containers/new/steps/LinkUserSelector.component.tsx

```diff
@@ -63,22 +65,25 @@ export default function LinkUserSelector({
   } = useGenerateS3Credentials({
     projectId,
     onSuccess: (credentials) => {
+      setS3Credentials(credentials);
       onSelectOwner({
         ...formUser,
+        access: credentials.access,
```

Comment:
React `set` function is asynchronous, so you have no guarantee that `credentials` has been updated when using it just above